packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -558,7 +558,8 @@ class SystemChrome {
   /// it can be hit-tested by the framework. On every frame, the framework will
   /// hit-test and select the annotated region it finds under the status and
   /// navigation bar and synthesize them into a single style. This can be used
-  /// to configure the system styles when an app bar is not used.
+  /// to configure the system styles when an app bar is not used. When an app
+  /// bar is used an annotated region is automatically created.
```

Comment:
Explain the combinations behind this. What happens when I set the system style _and_ there's an AppBar.

Reply:
+1